catalog/plugin/catalog-plugin-query-monitor-plugin/src/main/java/ddf/catalog/querymonitor/QueryMonitorPluginImpl.java

```diff
@@ -0,0 +1,170 @@
+/**
+ * Copyright (c) Codice Foundation
+ * <p>
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser
+ * General Public License as published by the Free Software Foundation, either version 3 of the
+ * License, or any later version.
+ * <p>
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without
+ * even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details. A copy of the GNU Lesser General Public License
+ * is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+package ddf.catalog.querymonitor;
+
+import java.util.HashMap;
+import java.util.UUID;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import ddf.catalog.operation.QueryRequest;
+
+import ddf.catalog.operation.QueryResponse;
+import ddf.catalog.plugin.PluginExecutionException;
+import ddf.catalog.plugin.PostFederatedQueryPlugin;
+import ddf.catalog.plugin.PreFederatedQueryPlugin;
+import ddf.catalog.plugin.StopProcessingException;
+import ddf.catalog.source.Source;
+
+public class QueryMonitorPluginImpl
+
+        implements QueryMonitorPlugin, PreFederatedQueryPlugin, PostFederatedQueryPlugin {
+
+    boolean removeFromTable = true;
+
+    private HashMap<UUID, ActiveSearch> activeSearchTable = new HashMap<UUID, ActiveSearch>();
```

Comment:
you should be able to use diamond operator
`private HashMap<UUID, ActiveSearch> activeSearchTable = new HashMap<>();`